test/test_ldap_connection.rb

```diff
@@ -340,6 +340,19 @@ def test_queued_read_bind_sasl
     assert result.success?
     assert_equal 2, result.message_id
   end
+
+  def test_invalid_pdu_type
+    options = {
+      app_tag: Net::LDAP::PDU::SearchResult,
```

Comment:
Is this required? not used in this test case.

Reply:
That's right!
I remove this parameter.